frontend/src/screens/wallet/WithdrawOnchainFunds.tsx

```diff
@@ -239,6 +262,36 @@ export default function WithdrawOnchainFunds() {
               }}
             />
           </div>
+          {info?.backendType === "LND" && (
+            <>
+              {showAdvanced && (
+                <div className="">
+                  <Label htmlFor="fee-rate">Fee Rate (Sat/Vbyte)</Label>
+                  <Input
+                    id="fee-rate"
+                    type="number"
+                    value={feeRate}
+                    required
+                    min={recommendedFees.minimumFee}
```

Comment:
Should we also add `max` with `recommendedFees.fastestFee` to save users?

Reply:
I don't think so, because it doesn't account for fee spikes. Would a confirmation would be better if the user chooses a rate larger than the fastest fee?

Reply:
Added confirmation alert

<img width="400" alt="Screenshot 2025-03-12 at 2 00 35 PM" src="https://github.com/user-attachments/assets/dae75828-4e31-4b92-97c4-6616b3e20d5b" />